### DISCUSSION TITLE
SPECFEM Discussions Tab

### DISCUSSION THREAD

**Discussion by bch0w** [SPECFEM/specfem3d#1548] [General]
I wanted to start a meta discussion/conversation about the SPECFEM organization Discussions tab, which we've agreed on to be the main forum for any future SPECFEM-related discussions. 

I just amended the default GitHub categories to include things like repository-level discussions (for 2D/3D/3D_Globe) and an ongoing research tab for people to show off cool things they're currently working on.

If Users have suggestions, comments or feedback on the discussions tab, e.g, how we might structure things better or specific topics that deserve they're on category, please feel free to drop a comment here!

**Comment by carltape** [comment]
[Here](https://docs.google.com/presentation/d/1PDK8Y74AJvZr7xKk_iYHXqp71BEbKMcy07nPIJ9lirE/edit?usp=sharing) are some google slide instructions for people to configure their accounts to receive email updates. Not sure if we should post these as a pdf or display in some other format. They need to be highly visible to new users as to how they can join the discussions.

**Reply by ljhwang** [reply]
Downloading a file can be another barrier. Can it be put on a web page?

**Reply by bch0w** [reply]
Good point, @ljhwang! @danielpeter has generated a .pdf from the slides and added it to the organization page. Hopefully all can see this link:
https://github.com/SPECFEM/.github/blob/main/docs/specfem_discussions.pdf

This PDF is also linked to on the main [specfem org page](https://github.com/SPECFEM).
